src/tmb/font.cpp

```diff
@@ -63,7 +63,20 @@ void fontSetCharWidth(int font, int char_width)
     fontInfo[font].char_width = char_width;
 }
 
-INCLUDE_ASM("asm/nonmatchings/tmb/font", fontSetDefaultSize__Fi);
+void fontSetDefaultSize(int font)
```

Comment:
```suggestion
void fontSetSpacing(int font, int spacing);
void fontSetDefaultSize(int font)
```

`fontSetDefaultSize()` is defined before `fontSetSpacing()` but calls into it, so we need to add `fontSetSpacing()`'s prototype somewhere above this function. (We can't move the actual definition of `fontSetSpacing()` or the binary won't match.)

It'll eventually be thrown into a `font.h` file, but for now we can just define it here.